crates/rolldown/src/utils/chunk/render_chunk_exports.rs

```diff
@@ -82,9 +82,10 @@ pub fn render_chunk_exports(
 
                 match export_mode {
                   Some(OutputExports::Named) => {
-                    if !options.external_live_bindings
+                    if (!options.external_live_bindings
                       && export_ref
-                        .is_created_by_import_from_external(&link_output.module_table.modules)
+                        .is_created_by_import_from_external(&link_output.module_table.modules))
+                      || options.experimental.is_disable_live_bindings_enabled()
```

Comment:
this could avoid analyze if it `is_disable_live_bindings_enabled`  is true
```rs
options.experimental.is_disable_live_bindings_enabled()  
 || export_ref.is_created_by_import_from_external(&link_output.module_table.modules))
 ```

Reply:
Done